src/main/kotlin/com/sourcegraph/cody/edit/FixupSession.kt

```diff
@@ -46,6 +47,10 @@ abstract class FixupSession(
     val project: Project,
     val document: Document
 ) : Disposable {
+  protected val documentBefore: Document =
```

Comment:
This isn't how Cody VSCode's diff view works. Diffing off this will include all your other edits you make in the file while Cody is working.

Cody VSCode [(source)](https://sourcegraph.com/github.com/sourcegraph/cody@c69aaf7/-/blob/vscode/src/non-stop/codelenses/provider.ts?L288-295) takes the file when you click "diff", re-inserts the original text we sent to the LLM in the range we're tracking for the replacement, and diffs that.

Reply:
@dominiccooney, I did it the same way now. can you please re-review?